src/diffusers/models/unet_2d_condition.py

```diff
@@ -105,15 +105,16 @@ def __init__(
             "CrossAttnDownBlock2D",
             "DownBlock2D",
         ),
-        mid_block_type: str = "UNetMidBlock2DCrossAttn",
+        mid_block_type: Optional[str] = "UNetMidBlock2DCrossAttn",
         up_block_types: Tuple[str] = ("UpBlock2D", "CrossAttnUpBlock2D", "CrossAttnUpBlock2D", "CrossAttnUpBlock2D"),
-        only_cross_attention: Union[bool, Tuple[bool]] = False,
+        only_cross_attention: Union[bool, Tuple[bool], None] = False,
```

Comment:
```suggestion
        only_cross_attention: Union[bool, Tuple[bool]] = False,
```

We should leave this as is I think :-)